autotx/utils/ethereum/helpers/fill_dev_account_with_erc20.py

```diff
@@ -0,0 +1,58 @@
+from decimal import Decimal
+from autotx.utils.ethereum import transfer_erc20
+from autotx.utils.ethereum.eth_address import ETHAddress
+from autotx.utils.ethereum.networks import NetworkInfo
+from eth_account.signers.local import LocalAccount
+from gnosis.eth import EthereumClient
+from autotx.utils.ethereum.uniswap.swap import build_swap_transaction
+
+
+def fill_dev_account_with_erc20(
+    client: EthereumClient,
+    dev_account: LocalAccount,
+    safe_address: ETHAddress,
+    network_info: NetworkInfo,
+) -> None:
+    tokens_to_transfer = {"usdc": 3500, "dai": 3500, "wbtc": 0.1}
```

Comment:
This will fill the account every time the user runs it
Can we do something like we check if the balance is lower than that amount? (like we do with eth)
Maybe even just top it up to that amount (and do it for native too) so that every run there is the specified amount of tokens in it.
We could even add a min threshold that triggers it instead of the treshold being the same as amount

Reply:
nice point... i will add a validation like we do with native